python_transport/wirepas_gateway/utils/argument_tools.py

```diff
@@ -166,6 +166,16 @@ def str2bool(value):
         else:
             raise argparse.ArgumentTypeError("Boolean value expected.")
 
+    @staticmethod
+    def str2int(value):
+        """ Ensures string to bool conversion """
+        try:
+            value = int(value)
+        except ValueError:
+            value = 0
```

Comment:
I would have keep the exception.

If you do a typo for port to 1883a for example, it will default the port to 0 and silently try a connection on this port...

Reply:
Good point. I'll add it back as it is keeps things more predictable.